examples/timer_gpio/timer_gpio_main.c

```diff
@@ -184,7 +184,6 @@ static int timer_gpio_daemon(int argc, char *argv[])
   notify.event.sigev_notify = SIGEV_SIGNAL;
   notify.event.sigev_signo  = CONFIG_EXAMPLES_TIMER_GPIO_SIGNO;
   notify.event.sigev_value.sival_ptr = NULL;
-  notify.oneshot = false;
```

Comment:
I think `oneshot` was renamed to `periodic`.

Reply:
It's already here https://github.com/apache/nuttx-apps/blob/c57e7a7b81d373c64b2a31708f68661e7f717b5b/examples/timer_gpio/timer_gpio_main.c#L182